app/src/controllers/groupOperations/reducer.js

```
@@ -52,70 +52,67 @@ const selectAll = (state, items) => {
-};
+    }
+    switch (type) {
+      case SET_VALIDATION_ERRORS:
+        return { ...payload };
+      case REMOVE_VALIDATION_ERRORS: {
+        const errorIds = Object.keys(state);
+        const payloadIds = payload.map(String);
+        return errorIds
+          .filter((id) => payloadIds.indexOf(id) === -1)
+          .reduce(
+            (result, key) => ({ ...result, [key]: state[key] }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)